modules/perception/fusion/lib/gatekeeper/pbf_gatekeeper/pbf_gatekeeper.cc

```diff
@@ -80,15 +79,9 @@ bool PbfGatekeeper::AbleToPublish(const TrackPtr &track) {
   struct tm timeinfo;
   localtime_r(&rawtime, &timeinfo);
   bool is_night = (timeinfo.tm_hour >= 23);
-  if (LidarAbleToPublish(track)) {
-    able_to_pub = true;
-  } else if (RadarAbleToPublish(track, is_night)) {
-    able_to_pub = true;
-  } else if (CameraAbleToPublish(track, is_night)) {
-    able_to_pub = true;
-  }
-
-  if (!able_to_pub) {
+  if (!LidarAbleToPublish(track) ||
```

Comment:
s/||/&&/ for it only fails when all sensor data fail to publish.

Reply:
I misunderstood the purpose before, thanks for your quick response!